<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1239</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="editor"/>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="paint">
        <property name="topMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLabel" name="error">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pageNumber"/>
        </item>
        <item>
         <widget class="FrameWidget" name="framewidget" native="true"/>
        </item>
        <item>
         <widget class="QListView" name="pagePreview">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>180</height>
           </size>
          </property>
          <property name="isWrapping" stdset="0">
           <bool>false</bool>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="gridSize">
           <size>
            <width>213</width>
            <height>150</height>
           </size>
          </property>
          <property name="viewMode">
           <enum>QListView::IconMode</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1239</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionCreatePDF"/>
    <addaction name="actionExport_PDF_as"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionLayoutTitle"/>
    <addaction name="actionLayoutBody"/>
    <addaction name="actionLayoutFullscreen"/>
    <addaction name="actionLayoutRight"/>
    <addaction name="actionLayoutLeft"/>
    <addaction name="separator"/>
    <addaction name="actionReload_Resources"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionCreatePDF"/>
   <addaction name="separator"/>
   <addaction name="actionLayoutTitle"/>
   <addaction name="actionLayoutBody"/>
   <addaction name="actionLayoutFullscreen"/>
   <addaction name="actionLayoutLeft"/>
   <addaction name="actionLayoutRight"/>
   <addaction name="actionTranslate"/>
   <addaction name="actionRotate"/>
  </widget>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionLayoutTitle">
   <property name="icon">
    <iconset theme="format-align-vertical-top">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="toolTip">
    <string>Title (Ctrl+T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionLayoutBody">
   <property name="icon">
    <iconset theme="menu_new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Body</string>
   </property>
   <property name="toolTip">
    <string>Body (Ctrl+B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionLayoutFullscreen">
   <property name="icon">
    <iconset theme="statetool">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="toolTip">
    <string>Fullscreen (Ctrl+U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionLayoutRight">
   <property name="icon">
    <iconset theme="bboxnext">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Right Column</string>
   </property>
   <property name="toolTip">
    <string>Right Column (Ctrl+R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLayoutLeft">
   <property name="icon">
    <iconset theme="bboxprev">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Left Column</string>
   </property>
   <property name="toolTip">
    <string>Left Column (Ctrl+L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionCreatePDF">
   <property name="icon">
    <iconset theme="pdflatex">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export PDF</string>
   </property>
   <property name="toolTip">
    <string>create PDF</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPresentationTitle">
   <property name="icon">
    <iconset theme="menu_new_sep">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Presentation Title</string>
   </property>
  </action>
  <action name="actionSubtitle">
   <property name="icon">
    <iconset theme="newline">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Subtitle</string>
   </property>
  </action>
  <action name="actionTranslate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="transform-move">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="object-rotate-right">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new-symbolic">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionReload_Resources">
   <property name="icon">
    <iconset theme="download">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Reload Resources</string>
   </property>
  </action>
  <action name="actionExport_PDF_as">
   <property name="icon">
    <iconset theme="document-export">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export PDF as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameWidget</class>
   <extends>QWidget</extends>
   <header location="global">framewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
